saleor/graphql/core/tests/test_core.py

```diff
@@ -248,14 +248,13 @@ def test_validate_image_file():
     img_data = BytesIO()
     image = Image.new("RGB", size=(1, 1))
     image.save(img_data, format="JPEG")
-    img = SimpleUploadedFile("product.jpg", img_data.getvalue(), "image/jpeg")
     field = "image"
 
     # when
-    result = validate_image_file(img, field, ProductErrorCode)
+    img = SimpleUploadedFile("product.jpg", img_data.getvalue(), "image/jpeg")
 
     # then
-    assert not result
+    validate_image_file(img, field, ProductErrorCode)
```

Comment:
`validate_image_file` is a method that we are testing not `SimpleUploadedFile`, so IMO it should go to `# when` part.

Reply:
The assertion is that calling this method does not raise an exception.

>When a valid JPEG file is uploaded, then `validate_image_file` does not raise.

Reply:
So from my perspective, we are checking if `validate_image_file` is not raising an error, so it's the method that we are testing. 🤔

Reply:
Ok, what is the test then? "When `validate_image_file` is called, then… `pass`?"

The old code checked if a function that never returns returned nothing.

Reply:
Ok, I get your point of view. We sometimes use `# when` and `# then` together, but it looks more clear.